app/views/home/vatReturnPeriod.scala.html

```diff
@@ -17,49 +17,78 @@
 @import views.html.main_template
 @import uk.gov.hmrc.play.views.html.layouts._
 @import models.UIHelpersWrapper
+@import uk.gov.hmrc.govukfrontend.views.html.components._
 
 @this(
-    mainTemplate: main_template,
-    uiHelpersWrapper: UIHelpersWrapper
+    layout: Layout,
+    uiHelpersWrapper: UIHelpersWrapper,
```

Comment:
I think we should get rid of this as well. I am doing it for other pages. Could remove any usage of it.